example3.js

```diff
@@ -0,0 +1,3 @@
+import { filename } from './index.js'
+
+console.log(filename(import.meta.url))
```

Comment:
Can you combine this example with the other ones? Maybe we can just have 1 example file.

Reply:
Sure. Addressed in commit 0f36d335716dc36f890636a1308c9938f6cfed8f.